pym/portage/package/ebuild/_config/LocationsManager.py

```diff
@@ -302,6 +302,7 @@ def set_root_override(self, root_overwrite=None):
 		self._check_var_directory("ROOT", self.target_root)
 
 		self.eroot = self.target_root.rstrip(os.sep) + self.eprefix + os.sep
+		self.broot = portage.const.EPREFIX
```

Comment:
But you're not using this at all?

Reply:
I'm not but @zmedico said it would be good to have it and we would later set it via the constructor.

Reply:
Maybe leave a TODO somewhere, so that people don't get too excited about using it.

Also, could you add something similar for sysroot in LocationsManager?

Reply:
I would add esysroot but until I don't know where to get the value from as the environment variables used in config.py (i.e. `mydict["ESYSROOT"]`) aren't available in this context. It may be easier when this gets rejigged around the constructor.